### PR TITLE
Vtouati pbi 221040 tocfixes

### PR DESCRIPTION
**Pull request by afloresOSI**
Here are the TOC fixes we discussed.

### CODE REVIEW

**Comment by GiselleLeeOSI** [comment]
@afloresOSI , it appears me and Josh's review are not needed. Please let us know if you think otherwise.

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 uid: transfer-data-to-ocs
 ---
 
-# Transfer PI System data to OCS
+# View PI System data fo transfer
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:5]: @VictoriaTouati-AVEVA , minor typo in this header; please correct.

File: content/rom/getting-started-rom.md [hunk 2]
```diff
@@ -10,17 +14,17 @@ The following procedure describes how to use the OCS Portal to monitor your asse
...
     The asset explorer displays the available assets. In this example, it provides an overview of the health of the fleet of trucks.
 
     Tip: Click on the icons in the toolbar to toggle between the card and table views of the assets.
     
     ![Card/Table View](images/AssetEditor_icons.png)
 
-1.  Scan through the assets and identify any assets with a problematic status.
+3. Scan through the assets and identify any assets with a problematic status.
 
     Tip: Use the Search box to filter the list of assets displayed.
```
Comment (rmuolic) [inline content/rom/getting-started-rom.md:29]: Bold "Tip:"

File: content/rom/getting-started-rom.md [hunk 3]
```diff
@@ -74,27 +78,27 @@ Procedure
...
     The trace is highlighted, and two cursors automatically mark the minimum and maximum values for the displayed time range.
 
     ![Maximum and minimum cursors](images/Max_min_cursors.png)
 
     Click the plus sign (+) above the trace to lock the cursor in place. The “+” turns into an “x.” Click the “x” to unlock the cursor.
 
-12.  Add other cursors at data points of interest.
+12. Add other cursors at data points of interest.
 
     Note: When two cursors are locked, the Legend table displays summary calculations for the values between the two cursors.
```
Comment (rmuolic) [inline content/rom/getting-started-rom.md:103]: Bold "Note:"

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/set-up-pi-to-ocs/confirm-agent-run.md [hunk 4]
```diff
@@ -22,8 +22,6 @@ After installation, confirm your PI to OCS Agent is running on the machine it wa
 
 **NOTE:** It may take a few minutes for your PI System to be registered. The following states may also appear next to the Data Source Status field to indicate a connection issue:
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/set-up-pi-to-ocs/confirm-agent-run.md:23]: **Note:**

**Review by rmuolic — COMMENTED** [review]
minor comments

**Comment by digitaljoshua** [comment]
i took a quick look at the commits regardless. looks  fine to me.

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md (hunk 1 above)
Comment (afloresOSI) [reply content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md]: Corrected

**Pull request merged**